src/sql/src/plan/statement/ddl.rs

```diff
@@ -1110,6 +1111,18 @@ fn kafka_sink_builder(
         bail!("exactly-once requires a consistency topic");
     }
 
+    if include_consistency {
+        for item in root_dependencies.iter() {
+            if item.item_type() == CatalogItemType::Source
+                && item.source_connector()?.yields_stable_input()
+            {
+                // it's all good
+            } else {
+                bail!("all transitive inputs of a consistent Kafka Sink must yield stable input, {} does not", item.name());
```

Comment:
Is there a way to word this in a simple way,  with words simpler than "transitive". Also, we do not seem to be defining "stable input" in our documentation.

Users, e.g. those coming from a DBA background would find all of this difficult to understand.

Reply:
I think this line is the hardest part in the PR... 🙈 

Maybe `all inputs of a consistent Kafka Sink must be replayable, {} is not`

The problem is the we don't yet have a term for this property of"replayable", or "stable", or "consistent" that we currently need for consistent sinks. @ruchirK any Ideas, because you're currently working on this from the input side?

Reply:
I pushed changes that split the error message and change the wording.

Reply:
Thank you!